pkg/helpers/image/test/util.go

```diff
@@ -21,6 +20,10 @@ import (
 	imagev1 "github.com/openshift/api/image/v1"
 )
 
+// Deprecated: Docker Image Manifest v2, Schema 1 is deprecated since 2015.
```

Comment:
hmm, this explains the above. Is there any reference for this?

Reply:
Yes, I found two references from official websites:

[CNCF distribution - current maintainer of distribution](https://distribution.github.io/distribution/spec/deprecated-schema-v1/)
[Docker - previous maintaner of distribution](https://docs.docker.com/engine/deprecated/#pushing-and-pulling-with-image-manifest-v2-schema-1)